src/LibVLCSharp.Forms/Shared/MediaPlayerElement.xaml

```diff
@@ -1,7 +1,53 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <ContentView xmlns="http://xamarin.com/schemas/2014/forms" 
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:controls="clr-namespace:LibVLCSharp.Forms.Shared"
+             xmlns:converter="clr-namespace:LibVLCSharp.Forms.Shared.Converters"
              x:Class="LibVLCSharp.Forms.Shared.MediaPlayerElement">
+    <ContentView.Resources>
```

Comment:
This whole diff should probably be in `Generic.xaml` instead, with the rest of the playback control styles.